src/main/java/net/openhft/chronicle/wire/YamlWire.java

```diff
@@ -1344,7 +1344,6 @@ public <T, K> WireIn sequence(@NotNull T t, K kls, @NotNull TriConsumer<T, K, Va
             while (true) {
                 switch (yt.current()) {
                     case SEQUENCE_ENTRY:
-                        yt.next(Integer.MIN_VALUE);
```

Comment:
what are these changes?

Reply:
This change and another was not needed for this feature